tests/test_builtins/en/test_usa_spec.py

```diff
@@ -47,6 +47,6 @@ def test_ssn(usa):
 
 
 def test_cpf_with_666_prefix(mocker, usa):
-    with mocker.patch.object(usa.random, 'randint', return_value=666):
```

Comment:
Tests fail on using `mocker` as a context manager. It happens because of the updated version of py.test plugins.